### PR TITLE
PR: Fix formating of text/documentation in tooltips and calltips

### LINKED ISSUE #9668: Hovers and calltips broken in the Editor
@dalthviz, I just found out that our hovers and calltips are broken in the Editor. These are some examples:

Selección_016

Selección_017

Selección_020

It seems adjusted things too much for the IPython console in PR #9585.

----

Please solve this one ASAP because we need to include it in beta3.

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [x] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Manage paragraphs wrapping instead of wrapping all the text for calltips, tooltips and hints
#### Editor and Console (Python)
mixins
#### Examples with other languages
other
imagen

### Issue(s) Resolved

Fixes #9668

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

**Comment by ccordoba12** [comment]
Please merge with master to get a new set of fixes for our tests.

**Comment by dalthviz** [comment]
A preview of the warnings and errors from the `Editor` (tooltip):

tooltip

**Comment by ccordoba12** [comment]
It looks much better, thanks @dalthviz! But in order to reduce the height of tooltips, let's take only 5 lines for each error/warning text returned by the PyLS (for instance, in the screenshot you uploaded the first two errors have 14 lines of text each, which I think is too much).

Besides, please apply a maximum height to tooltips of 25 lines, even if errors/warnings are left out.

**Comment by dalthviz** [comment]
A new preview:
toolt

File: spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py [hunk 1]
```diff
@@ -152,8 +152,29 @@ def close_console():
...
+          '''
+            <div style='font-family: "Consolas";
+                        font-size: 9pt;
+                        color: #999999'>
+                <br>Return evenly spaced values within a given '''
+          '''interval.<br><br>Values are generated within '''
+          '''the half-open interval ``[start, stop)``<br>'''), ])
 @pytest.mark.skipif(sys.platform == 'darwin', reason="Times out on macOS")
-def test_get_calltips(ipyconsole, qtbot):
+def test_get_calltips(ipyconsole, qtbot, function, signature, documentation):
```
Comment (ccordoba12) [inline spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py]: This is now failing on Linux, please take a look at it.

**Comment by ccordoba12** [comment]
> A new preview

Thanks @dalthviz!! Things are really cool now!

**Pull request merged**